src/ImportExport.ts

```diff
@@ -173,6 +224,12 @@ export const exportSynergism = async () => {
         : 'Savefile copied to file!';
 }
 
+export const errorGame = async () => {
```

Comment:
this function should be renamed to `preloadDeleteGame` or something closer to its actual purpose (deleting a save can be done just to start over or for testing, it doesn't necessarily have to be an issue with the game)

Reply:
It has changed
AWAIT could not detect the slow with try {} catch (), so I do not know the solution
It is good to warn you automatically, but you should ask if you should reset it